mods/tnt/init.lua

```diff
@@ -333,24 +333,24 @@ local function tnt_explode(pos, radius, ignore_protection, ignore_on_blast)
 	vm:update_liquids()
 
 	-- call nodeupdate for everything within 1.5x blast radius
+	for y = -radius * 1.5, radius * 1.5 do
 	for z = -radius * 1.5, radius * 1.5 do
 	for x = -radius * 1.5, radius * 1.5 do
-	for y = -radius * 1.5, radius * 1.5 do
 		local s = vector.add(pos, {x = x, y = y, z = z})
 		local r = vector.distance(pos, s)
```

Comment:
These two lines are equivalent to `local r = vector.length({x = x, y = y, z = z})`, which can be optimized to `local r = math.hypot(x, math.hypot(y, z))`.